app/modules/entities/components/layouts/edition.svelte

```diff
@@ -21,7 +22,11 @@
   const { uri, image, label } = entity
   let { claims } = entity
 
-  const claimsWithWorksClaims = _.pick(claims, filterClaims)
+  export const addWorksClaims = (claims, works) => {
+    const worksClaims = aggregateWorksClaims(works)
+    const nonEmptyWorksClaims = _.pick(worksClaims, isNonEmptyArray)
+    return Object.assign(claims, nonEmptyWorksClaims)
```

Comment:
So this is not a bug, since only non-edition properties are currently added (by `editionWorkProperties` only authors props and series prop). 
But, yes, it is safer this way. So its done in 75b9b6f53